CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ NOTES:
 BUGS FIXED:
 * Fix issue in Queries with vCD 10 version, which do not return network pool or provider VDC[#293](https://github.com/vmware/go-vcloud-director/pull/293)
 * Session timeout for media, catalog item upload  [#294](https://github.com/vmware/go-vcloud-director/pull/294)
+* Fix `vapp.RemoveNetwork`, `vapp.RemoveNetworkAsync` to use `DELETE` API call instead of update
```

Comment:
What about a comment that this is actual only for 10.1?

Reply:
It causes problems not only in 10.1. At least 10.0 and 9.5 also has problems with Terraform's binary test `vcd.TestAccVcdVAppVmMultiNIC.tf` reendering error similar to https://github.com/terraform-providers/terraform-provider-vcd/issues/473